owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -282,10 +290,38 @@ class PassCodeActivity : AppCompatActivity() {
         }
     }
 
+    private fun lockScreen() {
```

Comment:
I think this logic should be placed in the View Model instead of the Activity, you can observe when it finishes and notify the UI. What do you think?

Reply:
Ok good suggestion, I'll apply it 😀